livre/eq/Config.cpp

```diff
@@ -60,6 +97,65 @@ class Config::Impl
         , dataFrameRange( INVALID_FRAME_RANGE )
     {}
 
+    void gatherHistogram( const Histogram& histogram, const float area, const uint32_t currentId )
```

Comment:
I still think gather is not the correct verb. Gather is an active action, whereas this function is already receiving the histogram as an input parameter.